src/auth/screens/auth-profile.screen.js

```diff
@@ -139,74 +148,84 @@ class AuthProfile extends Component {
 
     return (
       <ViewContainer>
-        {loading && <LoadingContainer animating={loading} center />}
-
-        {!loading &&
-          <ParallaxScroll
-            renderContent={() =>
-              <UserProfile
-                type="user"
-                initialUser={user}
-                user={user}
-                navigation={navigation}
-              />}
-            stickyTitle={user.login}
-          >
-            {user.bio &&
-              user.bio !== '' &&
-              <SectionList title="BIO">
-                <ListItem
-                  subtitle={user.bio}
-                  subtitleStyle={styles.listSubTitle}
-                  hideChevron
-                />
+        <ScrollView
+          refreshControl={
+            <RefreshControl
+              refreshing={loading}
+              onRefresh={this.refreshProfile}
+            />
+          }
+        >
+          {loading && <LoadingContainer animating={loading} center />}
+
+          {!loading &&
+            <ParallaxScroll
+              renderContent={() =>
+                <UserProfile
+                  type="user"
+                  initialUser={user}
+                  user={user}
+                  navigation={navigation}
+                />}
+              stickyTitle={user.login}
+            >
+              {user.bio &&
+                user.bio !== '' &&
+                <SectionList title="BIO">
+                  <ListItem
+                    subtitle={user.bio}
+                    subtitleStyle={styles.listSubTitle}
+                    hideChevron
+                  />
               </SectionList>}
 
-            <EntityInfo entity={user} orgs={orgs} navigation={navigation} />
+              <EntityInfo entity={user} orgs={orgs} navigation={navigation} />
+
+              <SectionList
+                title="ORGANIZATIONS"
+                noItems={orgs.length === 0}
+                noItemsMessage={'No organizations'}
+              >
+                {orgs.map(item =>
+                  <UserListItem
+                    key={item.id}
+                    user={item}
+                    navigation={navigation}
+                  />
+                )}
+                <Text style={styles.note}>
+                  Can&apos;t see all your organizations?{'\n'}
```

Comment:
I thought it was already wrapped in that component? It starts at line 196. Unless I am missing something or not catching what you meant?